docs/api/server/Environment-Variables.md

```diff
@@ -714,6 +725,86 @@ ONE_SERVICE_WORKER=true
 ONE_SERVICE_WORKER=false
 ```
 
+## `OTEL_EXPORTER_OTLP_LOGS_ENDPOINT`
```

Comment:
what about noting that If this is enabled the other log arguments are no longer considered ? eg `logFormat`

Reply:
added in 29f0b97550d0adf1d23e2f68015c19bcf50f5dd1